modules/profile/templates/initscripts/matterbridge.systemd.erb

```diff
@@ -0,0 +1,19 @@
+# Managed By Puppet
+[Unit]
+Description=Chat relay bot
+Documentation=https://github.com/42wim/matterbridge
+After=network-online.target
+
+[Service]
+Type=simple
+WorkingDirectory=/discord-irc
```

Comment:
Can it not run out of its own dir?

preferably in /srv

Reply:
sure